### PR TITLE
Training Title Bar notification

### PR DESCRIPTION
**Pull request by MiteyDrill**
-Webpage Title blinks when user training finishes
 -Unsure if this works in all cases (jutsu training, etc)

### CODE REVIEW

**Comment by MiteyDrill** [comment]
Need to change the 995 to 1000, if number not changed larger training time will cause the notification to start too early on AFK webpage tabs

File: classes/user.php [hunk 1]
```diff
@@ -844,36 +844,40 @@ public function loadData($UPDATE = 2, $remote_view = false) {
...
+          setTimeout(()=>{titleBarFlash();}, train_time * 995);
 					</script>";
 				}
 				else  {
+          //*setTimeout is used to notify training finished*//
 					$display .= "<p class='trainingNotification'>Training: " . ucwords(str_replace('_', ' ', $this->train_type)) . "<br />" .
 						"<span id='trainingTimer'>" . timeRemaining($this->train_time - time(), 'short', false, true) . " remaining</span></p>";
 					$display .= "<script type='text/javascript'>
 					let train_time = " . ($this->train_time - time()) . ";
+          setTimeout(()=>{titleBarFlash();}, train_time * 995);
```
Comment (levimeahan) [inline classes/user.php:875]:
```suggestion
          setTimeout(()=>{titleBarFlash();}, train_time * 1000);
```

File: classes/user.php [hunk 2]
```diff
@@ -844,36 +844,40 @@ public function loadData($UPDATE = 2, $remote_view = false) {
...
 			else {
 				require_once("functions.php");
+        //*setTimeout is used to notify training finished*//
 				if(strpos($this->train_type, 'jutsu:') !== false) {
 					$train_type = str_replace('jutsu:', '', $this->train_type);
 					$display .= "<p class='trainingNotification'>Training: " . ucwords(str_replace('_', ' ', $train_type)) . "<br />" .
 					"<span id='trainingTimer'>" . timeRemaining($this->train_time - time(), 'short', false, true) . " remaining</span></p>";
 					$display .= "<script type='text/javascript'>
 					let train_time = " . ($this->train_time - time()) . ";
+          setTimeout(()=>{titleBarFlash();}, train_time * 995);
```
Comment (levimeahan) [inline classes/user.php:866]:
```suggestion
          setTimeout(()=>{titleBarFlash();}, train_time * 1000);
```

File: scripts/functions.js [hunk 3]
```diff
@@ -5,12 +5,19 @@ function selectAllMessages() {
 var currentNotification = 0;
 var totalNotifications = 0;
 
+function titleBarFlash() {
+	setInterval(()=>
+	{
+		document.title = document.title == "Training Done!" ? "" : "Training Done!";
```
Comment (levimeahan) [inline scripts/functions.js:11]:
```suggestion
		document.title = document.title == "Training Done!" ? "Shinobi Chronicles RPG" : "Training Done!";

```

**Review by levimeahan — APPROVED** [review]

**Pull request merged**